zerver/lib/user_groups.py

```diff
@@ -627,6 +630,9 @@ def user_groups_in_realm_serialized(
             can_mention_group=get_setting_value_for_user_group_object(
                 user_group.can_mention_group, group_members, group_subgroups
             ),
+            can_remove_members_group=get_setting_value_for_user_group_object(
+                user_group.can_remove_members_group, group_members, group_subgroups
+            ),
```

Comment:
This function is probably getting to the point where using a `*group_kwargs` pattern generated using a loop over the settings might be nicer.

Reply:
Posted about this - https://github.com/zulip/zulip/pull/32564#issuecomment-2517418190